protos/feast/core/Store.proto

```diff
@@ -110,6 +110,8 @@ message Store {
     int32 max_retries = 4;
     // Optional. How often flush data to redis
     int32 flush_frequency_seconds = 5;
+
+    bool ssl = 6;
```

Comment:
You are missing a comment, and should this be `enable_ssl`?

Reply:
Added a comment. This parameter is called just `ssl`/`setSsl` both in python and java client so I left it that way for consistency